includes/class-database-cleanup.php

```
@@ -143,19 +161,45 @@ public static function clean_revisions_advanced( $max_age_days = 30, $keep_lates
+
+			if ( $posts_deleted === false ) {
+				return false;
+			}
+
+			if ( $posts_deleted ) {
+				$deleted += (int) $posts_deleted;
+			}
+		} while ( count( $post_ids ) >= 1000 );
+
+		return $deleted;
 	}
```

PHPMD (2.15.0)

[error] The method clean_auto_drafts is not named in camelCase. (undefined)

(CamelCaseMethodName)